docs/cn/sitemap.xml

```diff
@@ -1,198 +1,188 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-
-    
-        
     <url>
-     <loc>/about-appium/intro/</loc>
+     <loc>None</loc>
```

Comment:
should it be none?

Reply:
mm, probably something was missed from 0.17 to 1.0. Will take a look